### PR TITLE
Cramorant always changes to Cramorant-Gorging if its HP is 50% or less

### PR DESCRIPTION
**Pull request by KrisXV**

### CODE REVIEW

File: data/abilities.js [hunk 1]
```diff
@@ -1373,7 +1373,7 @@ let BattleAbilities = {
 			if (!['surf', 'dive'].includes(move.id) || source.volatiles['dive'] || source.speciesid !== 'cramorant' || source.transformed) return;
 
 			// PLACEHOLDER Estimated 10% of the time you get pikachu
-			const forme = this.random(10) === 1 ? 'cramorantgorging' : 'cramorantgulping';
+			const forme = this.random(10) === 1 || source.hp <= Math.floor(source.maxhp / 2) ? 'cramorantgorging' : 'cramorantgulping';
```
Comment (TheImmortal) [inline data/abilities.js]: I don't understand. If it's 50%, why did you keep the 10% roll?

Comment (KrisXV) [reply data/abilities.js]: it still has a random chance to change into Cramorant-Gorging above 50%

Comment (HoeenCoder) [reply data/abilities.js]: Some tests that my brother did for me confirms its possible to get Gorging above 50%. Going to merge as is and get the exact percent later.

Comment (HoeenCoder) [reply data/abilities.js]: After testing about 50 uses of surf cramorant with > 50% HP, Pikachu never appeared. So I'd say its supposed to be the fish when > 50%, and pikachu when < 50%.

File: data/abilities.js [hunk 2]
```diff
@@ -1371,9 +1371,7 @@ let BattleAbilities = {
 		},
 		onAfterMove(source, target, move) {
 			if (!['surf', 'dive'].includes(move.id) || source.volatiles['dive'] || source.speciesid !== 'cramorant' || source.transformed) return;
-
-			// PLACEHOLDER Estimated 10% of the time you get pikachu
-			const forme = this.random(10) === 1 ? 'cramorantgorging' : 'cramorantgulping';
+			const forme = source.hp <= Math.floor(source.maxhp / 2) ? 'cramorantgorging' : 'cramorantgulping';
```
Comment (urkerab) [inline data/abilities.js]: Note that Belly Drum doesn't bother using `Math.floor`.

Comment (HoeenCoder) [reply data/abilities.js]: Any reason this was marked as resolved?

Comment (KrisXV) [reply data/abilities.js]: I dont remember doing this

Comment (KrisXV) [reply data/abilities.js]: ok now it is actually resolved

**Pull request merged**